src/components/search-form/search-form.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './search-form.css'
+
+const SearchForm = ({ initialValue, onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState(initialValue || '');
+  const placeholder = 'What do you want to watch?';
+
+  const handleInputChange = () => {
+    onSearch(searchTerm);
+  };
+
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
+        onSearch(searchTerm);
+    }
+  };
+
+  return (
+    <div className='search-form'>
```

Comment:
Any time you see inputs and submits - makes sense to use a form. In fact, if you used a form, you wouldn't have to listen for key events - forms handle enter submit automatically. So you would just need to create a submit button and an onSubmit handler for the form

Reply:
Replaced outer div with form and added onSubmit handler